be/src/runtime/decimalv3.h

```diff
@@ -189,13 +189,25 @@ class DecimalV3Cast {
     }
 
     template <typename T>
-    static constexpr T float_overflow_indicator = std::numeric_limits<T>::lowest();
+    static constexpr T float_lower_overflow_indicator = std::numeric_limits<T>::max();
```

Comment:
for double and float, std::numeric_limits<double>::lowest() is the actual minimum.
https://en.cppreference.com/w/cpp/types/numeric_limits

Reply:
The actual type of decimal, which we compare with here, is integer.